{{cookiecutter.repo_name}}/package.json

```diff
@@ -11,34 +11,36 @@
     "tinify": "gulp tinify"
   },
   "author": "{{ cookiecutter.author }}",
-  "dependencies": {
-    "babel-core": "^6.24.1",
-    "babel-loader": "^7.0.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-es2017": "^6.24.1",
-    "del": "^2.2.2",
-    "gulp": "^3.9.1",
-    "gulp-clean-css": "^3.0.4",
-    "gulp-header": "^1.8.8",
-    "gulp-plumber": "^1.1.0",
-    "gulp-sourcemaps": "^2.6.0",
-    "gulp-suitcss": "^3.0.0",
-    "gulp-tinify": "^1.0.2",
-    "gulp-uglify": "^2.1.2",
-    "postcss-nested": "^1.0.1",
-    "run-sequence": "^1.2.2",
-    "suitcss": "^3.0.0",
-    "webpack": "^2.3.3",
-    "webpack-stream": "^3.2.0"
-  },
   "devDependencies": {
```

Comment:
I'm curious why you made these dev dependencies vs normal dependencies?

Reply:
The top highlight on here explains it pretty well:
https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c
> The difference between these two, is that devDependencies are modules which are only required during development, while dependencies are modules which are also required at runtime.
(so devDependencies is for build process, environment, etc. whereas dependencies would include React, Glidejs, Fitty, etc.)